src/app/dashboard/error.tsx

```
@@ -0,0 +1,47 @@
+  }, [error]);
+
+  return (
+    <div className="flex min-h-[600px] flex-col items-center justify-center gap-4 px-4">
+      <AlertCircle className="h-16 w-16 text-red-500" />
+      <div className="text-center">
+        <h2 className="mb-2 text-2xl font-bold text-gray-900 dark:text-gray-100">
+          Dashboard Error
+        </h2>
+        <p className="mb-6 max-w-md text-gray-600 dark:text-gray-400">
+          Something went wrong while loading the dashboard. This has been
+          logged and we'll look into it.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&#39;`, `&rsquo;`.

(react/no-unescaped-entities)